hackIDE_project/settings.py

```diff
@@ -23,11 +23,10 @@
 SECRET_KEY = '+h*i@$52+w(_e#etvzgnkjq!q0ajz1qpgs-y9%89x4w3nlct=m'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
-#(os.environ.get('HACKIDE_DEBUG') != None)
+DEBUG = (os.environ.get('HACKIDE_DEBUG') != None)
 # DEBUG = (os.environ.get('HACKIDE_DEBUG') or "").lower() == "true"
 
-ALLOWED_HOSTS = ['hackide.herokuapp.com'] if not DEBUG else ['*']
+ALLOWED_HOSTS = ['hackide.herokuapp.com'] if DEBUG else ['*']
```

Comment:
This is wrong. The actual logic is - 

Allow on hackide.herokuapp.com only, if NOT DEBUG (Production)
Allow everywhere, if DEBUG (localhost mostly)

So, according to this, the previous logic should work.